langchain/src/tools/convert_to_openai.ts

```diff
@@ -11,7 +12,9 @@ import { StructuredTool } from "./base.js";
  */
 export function formatToOpenAIFunction(
   tool: StructuredTool
-): OpenAIClient.Chat.ChatCompletionCreateParams.Function {
+):
+  | OpenAIClient.Chat.ChatCompletionCreateParams.Function
+  | AssistantCreateParams.AssistantToolsFunction {
```

Comment:
What's the difference between these two types?

Should probably be two different methods